src/components/OSMElementData/OSMElementData.jsx

```diff
@@ -0,0 +1,186 @@
+import { Fragment, useState, useEffect } from 'react'
```

Comment:
I feel this file is way more verbose than necessary. But that could just be "because react", please suggest any improvement here.

Reply:
With `useQuery` it looks much cleaner.